packages/dev/core/src/FlowGraph/Blocks/Data/flowGraphCachedOperationBlock.ts

```diff
@@ -0,0 +1,35 @@
+import type { IFlowGraphBlockConfiguration } from "../../flowGraphBlock";
+import { FlowGraphBlock } from "../../flowGraphBlock";
+import type { FlowGraphContext } from "../../flowGraphContext";
+import type { FlowGraphDataConnection } from "../../flowGraphDataConnection";
+import type { RichType } from "../../flowGraphRichTypes";
+
+const CACHE_NAME = "cachedOperationValue";
+const EXEC_ID_NAME = "executionId";

```

Comment:
I would call it `cachedExecutionId` to make clear it is the executionId value related to `cachedOperationValue`.